### ISSUE TITLE
macOS Ventura

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#725]
Should be released today.
Rumor on the street is resize problems in Logic and nothing else.

**Comment by p0nce** [comment]
So far it seems only `altool` invocation doesn't work and fail with errorcode 1. Probably, the older notarization method is deprecated.

**Comment by p0nce** [comment]
- [ ] AU validation fail on Ventura
```
FORMAT TESTS:

Reported Channel Capabilities (explicit):
      [1, 1]  [2, 2]  

Input/Output Channel Handling:
1-1   1-2   1-4   1-5   1-6   1-7   1-8   2-2   2-4   2-5   2-6   2-7   2-8   4-4   4-5   5-5   6-6   7-7   8-8
X                                         X                                                                       
ERROR: -10879 IN CALL Cannot Set Output Num Channels:2 when unit says it can
ERROR: -10879 IN CALL Cannot Set Output Num Channels:2 when unit says it can
```

**Comment by p0nce** [comment]
OK the notarization problem was Apple revoking my app-specific-password, probably because the same was used for both stapler and altool; or they expired.
- [ ] Instruct to use different password for both, altool and stapler
- [ ] -v flag should pass to altool and stapler as --verbose

**Comment by p0nce** [comment]
So.... the existing plugins validate in current logic whether or not .rsrc is there.
No Dplug plugin validate when `auvaltool` is used with `-oop` flag.
So there is still a risk of future Logic crashing our plugins.

(One AU bug was fixed in #621 but unrelated.)

**Comment by p0nce** [comment]
Okay, we have Ventura compat
